src/main/resources/views/CommercialVehicles/failWelsh.hbs

```diff
@@ -70,7 +70,7 @@
         </div>
         <div class="footer__certificate-tag">
                <span class="footer__certificate-tag-text">
-                   {{presentedDocumentNameFail}}/{{versionNumberFail}}
+               {{presentedDocumentNameFailWelsh}}/{{VersionNumberFailWelsh}}
```

Comment:
I've just generated a certificate and the following is being returned:

```
Tudalen 1 o 2
VTP30W/
```

It's missing the version number, I believe if it is because you have capitalised V in versionNumberFailWelsh. Can you re-test to validate once you have made the change, built/deployed please.